share/shwap/p2p/shrex/shrexeds/pb/extended_data_square.proto

```diff
@@ -1,8 +1,6 @@
 syntax = "proto3";
 
-message EDSRequest {
-  bytes hash = 1; // identifies the requested EDS.
-}
+option go_package = "github.com/celestiaorg/celestia-node/share/shwap/p2p/shrex/shrexeds/pb";
```

Comment:
After https://github.com/libp2p/specs/pull/623, we won't need this PB just for error codes anymore

Reply:
We would still need a way to differentiate internal errors from rate limiting.

Reply:
which is possible with error codes